src/widgets/SurveyAnswerProcessing/ui/ProgressBar.tsx

```diff
@@ -0,0 +1,44 @@
+import { CheckCircle } from '../../../shared/ui';
+
+import { Theme } from '~/shared/constants';
+import Box from '~/shared/ui/Box';
+import Loader from '~/shared/ui/Loader';
+import Text from '~/shared/ui/Text';
+
+type Props = {
+  isNotStarted: boolean;
+  isCompleted: boolean;
+  isInProgress: boolean;
+  activityName: string;
+  currentActivityIndex: number;
+  activitiesCount: number;
+};
+
+export const ProgressBar = ({
+  currentActivityIndex,
+  activitiesCount,
+  activityName,
+  isCompleted,
+}: Props) => {
+  return (
+    <Box
+      display="flex"
+      borderRadius="12px"
+      padding="16px 8px"
+      marginTop="16px"
```

Comment:
margins and paddings should be applied at the upper level, not inside the component

Reply:
Done